.gitignore

```diff
@@ -48,6 +48,8 @@ nips2010_pdf/
 *.tgz
 
 examples/cluster/joblib
+examples/release_highlights/joblib
```

Comment:
the `joblib` folder appears where you run the script from, not where the script is located it seems

Reply:
Sorry, I don't understand the comment... :(
 I'm not yet really into joblib things so I don't know which building process generated the `joblib` directory... Those two directories show up after a documentation build `make html` in the `doc` directory... is that useful?

Reply:
I meant that if you run `python scripy.py` from `my_folder` you will have `my_folder/joblib`.

It won't always be in ` examples/release_highlights/joblib `, that's just where sphinx-gallery runs the example from.

But... That's not relevant anyway ^^


The `joblib` folder is created because we use `    Isomap(n_neighbors=10, metric='precomputed'), memory='.'`

@TomDLT could we use e.g. `/tmp`?

Reply:
Sure !

Reply:
Would that work on Windows? We might want to use `tempfile.gettempdir()` instead.